Automattic/simplenote-ios
# Fixing StatusBar Color

### Fix
In this PR we're fixing a (super simple) bug that prevented the StatusBar from being visible, whenever Simplenote's Settings are OnScreen (iOS 13 only).

@etoledom Sir, may I bug you with a super small PR?

Thanks in advance!!

Closes #456 

### Test
1. Launch Simplenote in an iOS 13 device
2. Press the top left button to open the Tags List
3. Press over the Settings row

- [x] Verify that the StatusBar is visible

### Release
`RELEASE-NOTES.txt` was updated in c03e311 with:

> Fixed a bug that caused the StatusBar to be hidden when Simplenote's Settings were displayed
